noxfiles/docker_nox.py

```diff
@@ -9,35 +15,36 @@ def get_current_image() -> str:
 
 
 @nox.session()
-def build(session: nox.Session) -> None:
+@nox.parametrize(
+    "image",
+    [
+        nox.param("prod", id="prod"),
+        nox.param("dev", id="dev"),
+        nox.param("test", id="test"),
+        nox.param("ui", id="ui"),
+    ],
+)
+def build(session: nox.Session, image: str) -> None:
     """Build the Docker container for fidesctl."""
+    build_matrix = {
+        "prod": {"tag": get_current_image(), "target": "prod"},
```

Comment:
good call, this function getting called requires the git module to be installed, hence why this is happening...i need to fix this so that it doesn't get called unless its actually getting used